frontend/src/features/admin-form/settings/components/PaymentSettingsSection/PaymentSettingsSection.tsx

```diff
@@ -194,33 +198,45 @@ const PaymentsAccountInformation = ({
 }
 
 export const PaymentSettingsSection = (): JSX.Element => {
-  const { hasPaymentCapabilities, data } = useAdminFormPayments()
-  const stripeAccount = data?.account
+  const {
+    hasPaymentCapabilities,
+    isLoading: adminFormPaymentsLoading,
+    isError: adminFormPaymentsError,
+  } = useAdminFormPayments()
 
   const { data: settings, isLoading: settingsIsLoading } =
     useAdminFormSettings()
   const { data: { secretEnv } = {} } = useEnv()
   const isProductionEnv = secretEnv === 'production'
 
   return settings?.responseMode === FormResponseMode.Encrypt ? (
-    !stripeAccount ? (
-      <BeforeConnectionInstructions isProductionEnv={isProductionEnv} />
-    ) : (
-      <Skeleton isLoaded={!settingsIsLoading}>
-        <AfterConnectionInfo isProductionEnv={isProductionEnv} />
-        <PaymentsAccountInformation
-          account_id={settings?.payments_channel?.target_account_id}
-          isLoading={settingsIsLoading}
-        />
-        <StripeConnectButton connectState={StripeConnectButtonStates.LINKED} />
-        {hasPaymentCapabilities && (
-          <>
-            <Divider my="2.5rem" />
-            <BusinessInfoSection />
-          </>
-        )}
-      </Skeleton>
-    )
+    <Skeleton isLoaded={!settingsIsLoading}>
+      {settings.payments_channel.channel === PaymentChannel.Unconnected ? (
+        <BeforeConnectionInstructions isProductionEnv={isProductionEnv} />
+      ) : (
+        <Skeleton isLoaded={!adminFormPaymentsLoading}>
+          <AfterConnectionInfo
+            isProductionEnv={isProductionEnv}
+            hasPaymentCapabilities={hasPaymentCapabilities}
+            adminFormPaymentsLoading={adminFormPaymentsLoading}
+            adminFormPaymentsError={adminFormPaymentsError}
```

Comment:
When `adminFormPaymentsLoading` is visible to the user and passed in here, it is guaranteed to be `false`. Otherwise the skeleton won't load.

We can remove this property.

Reply:
Done in [34dc6e1](https://github.com/opengovsg/FormSG/pull/6390/commits/34dc6e16d3c39fcbd8df095f4fef98810bd836c8)